### DISCUSSION TITLE
Copilot for internal-dev team

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#5391] [General]
Hey @kodadot/internal-dev I would like help you in any way possible to be efficient, to speedup some easy efforts and crack harder tasks easier, I was thinking that I would be happy repay you Copilot license for let's say upcoming 3 months! wyt?

If so, just let me know and I'll send you extra bucks for this for your existing address.

https://github.com/features/copilot#pricing

**Comment by daiagi** [comment]
so far i found it not so useful. chatGPT4 is far more useful even without integration into the IDE
I'll be more interested when CopilotX hits the market
